rllib/agents/impala/vtrace_tf_policy.py

```diff
@@ -242,15 +242,6 @@ def grad_stats(policy, train_batch, grads):
     }
 
 
-def postprocess_trajectory(policy,
-                           sample_batch,
-                           other_agent_batches=None,
-                           episode=None):
-    # not used, so save some bandwidth
-    del sample_batch.data[SampleBatch.NEXT_OBS]
-    return sample_batch
```

Comment:
Can we keep this? What would happen if a user had this in a custom copy of Impala?

Reply:
keeping it, however, in the test run, this will be a TrackingDict, not a SampleBatch, so it won't have the `data` prop.
I had to add a key check.

Reply:
How about adding this to TrackingDict for backwards compat?
```
@property
def data(self):
   return self   # backwards compat with SampleBatch
```
I really want to make sure there are zero lines of code change in the policy files.

Reply:
sounds good.

Reply:
done

Reply:
This doesn't seem fixed